LayoutTests/http/tests/workers/reportingObserverNonNull.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+<script src="../resources/common.js"></script>
+</head>
+<body>
+<p id="description"></p>
+<div id="console"></div>
+<script>
+  description("Test generating a ReportingObserver. PASSING if not crashing");
+  new Image().src = 'data:';
+  new Worker(`data:text/javascript,new ReportingObserver(() => {}).observe();`);
+  finishJSTest();
```

Comment:
I bet this test needs to be async since the worker doesn't get constructed synchronously?
Calling `finishJSTest()` without setting `window.jsTestIsAsync` to true is not correct.

Also, you likely need to call `finishJSTest()` when the Worker is done running. Or if you verified that your test was able to reproduce the crash as-is (in WebKitTestRunner, not in browser), then you can drop finishJSTest() like Youenn suggested (but I personally doubt it).